Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM dlang2/dmd-ubuntu
+LABEL maintainer="Silq Team"
+LABEL description="Silq is a high-level programming language for quantum computing with a strong static type system."
+
+RUN apt update
+RUN apt install -y git ldc
```

Comment:
Is there a reason this installs ldc? (`dependencies-release.sh` automatically fetches a compatible version of ldc into a local folder.)

Reply:
Hi @tgehr — sorry for the delay here! I tried originally without this explicit installation line and it looked like the install was missing `ldmd2`:

```
./build-release.sh: line 18: ldmd2: command not found
```

I will confess that I'm completely new to Dlang so it's possible that this is just an alias for a tool that I _do_ have in the container... I'm just not quite sure :) With the `ldc` install added here, it does indeed succeed.